frontend/package.json

```diff
@@ -17,6 +18,7 @@
   },
   "scripts": {
     "build": "routify -b && cross-env NODE_ENV=production webpack",
+    "buildDev": "routify -b && cross-env NODE_ENV=development webpack",
```

Comment:
I dont really care exactly how we handle this; I just need `build` locally to call from hot reloading. this works fine